### PR TITLE
fix: Prevent double jailing

### PR DESCRIPTION
**Pull request by gr8h**
This PR resolves a bug where a validator can be erroneously listed multiple times in the `JAILED_VALIDATORS` array.

Steps to Reproduce:
1. A validator initiates a maintenance transaction.
2. The emitInitiateChange function is subsequently called.
3. The validator is listed twice in the `JAILED_VALIDATORS` list.

This PR is building upon: https://github.com/fuseio/fuse-network/pull/146/files

### CODE REVIEW

File: scripts/5_validator_tx.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
+require("dotenv").config();
```
Comment (leonprou) [inline scripts/5_validator_tx.js:1]: @gr8h 
can you rename the script and add comments to explain the purpose of the script

**Review by leonprou — CHANGES_REQUESTED** [review]
lgtm
see the comments

File: scripts/5_validator_tx.js (hunk 1 above)
Comment (gr8h) [reply scripts/5_validator_tx.js]: renamed to `5_validator_operations.js`

**Review by leonprou — APPROVED** [review]
LGTM

**Pull request merged**